app/controllers/reservations_controller.rb

```diff
@@ -330,7 +330,7 @@ def checkout # rubocop:disable all
     # prep for receipt page and exit
     @check_in_set = []
     @check_out_set = checked_out_reservations
-    render('receipt') && return
+    render('receipt', layout: 'application_with_search_sidebar') && return
```

Comment:
I think that design wise, it looks a bit awkward since the only thing we have in the sidebar is that little search box. I'm not sure if adding in the functionality to search for users on all pages on the ReservationsCatalog views, justifies using up 1/4 of the space. So I'm leaning towards only showing it for the receipt as on that page it directly affects the workflow.